### DISCUSSION TITLE
Check my totp module

### DISCUSSION THREAD

**Discussion by OdaiGH** [vlang/v#19884] [General]
check my new module for v lang

https://github.com/OdaiGH/votp

**Comment by JalonSolov** [comment]
Nice.  Do you plan to add this to the Awesome V page?  https://github.com/vlang/awesome-v

**Reply by OdaiGH** [reply]
i dont know how 

however i just updated it and added hotp

**Reply by JalonSolov** [reply]
It is simple to update that page... clone the repo, edit the README.MD file, and submit a Pull Request.
